src/components/Profile/components/PersonalInfoList/index.jsx

```diff
@@ -55,6 +59,10 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
     AuthGroup.SensitiveInfoView,
     AuthGroup.AcademicInfoView,
   );
+  const [isJoinDialogOpen, setIsJoinDialogOpen] = useState(false);
+  const onClose = () => {
+    setIsJoinDialogOpen(false);
+  };
```

Comment:
```suggestion
  const onClose = () => {
    setIsJoinDialogOpen(false);
  };
```

`onClose` is called only once. You should probably just set the state from the location directly